### ISSUE TITLE
ArgumentException in Unity in OVRProjectConfig

### ISSUE THREAD

**Issue by mludlum** [oculus-samples/Unity-CrypticCabinet#10]
Can't get this to compile on 2022.3.42f1.git.3001194.   Try to get this running on a Quest 3.   I followed the project setup instructions w.r.t. app setup on oculus.com.    I updated the packages.   I'm getting this error in the Console.   

ArgumentException: Path cannot be the empty string or all whitespace.
Parameter name: path
System.IO.Directory.GetParent (System.String path) (at <8ce0bd04a7a04b4b9395538239d3fdd8>:0)
OVRProjectConfig.ComputeOculusProjectAssetPath (System.String assetName) (at ./Library/PackageCache/com.meta.xr.sdk.core@67.0.0/Editor/OVRProjectConfig.cs:223)
OVRProjectConfig.ComputeOculusProjectConfigAssetPath () (at ./Library/PackageCache/com.meta.xr.sdk.core@67.0.0/Editor/OVRProjectConfig.cs:246)
OVRProjectConfig.GetOrCreateProjectConfig () (at ./Library/PackageCache/com.meta.xr.sdk.core@67.0.0/Editor/OVRProjectConfig.cs:252)
OVRProjectConfig.Update () (at ./Library/PackageCache/com.meta.xr.sdk.core@67.0.0/Editor/OVRProjectConfig.cs:213)
UnityEditor.EditorApplication.Internal_CallUpdateFunctions () (at <fd823a7663ea4cdb840e6c898e2256ce>:0)

**Comment by jasonmeisel** [comment]
That issue is very odd! For more context, what do you mean by "2022.3.42f1.git.3001194"? Are you using a publicly available build of Unity?

**Comment by mludlum** [comment]
It was what Help->About showed.  It was a publicly available build.

**Comment by AlexThiv** [comment]
@mludlum can you confirm that you cloned the repository using lfs?
If you are still running into this issue, can you either reimport all or Close Unity delete the Library folder and reopen the project.
We are looking into this section of the code and it seems like either the engine has a hard time figuring out the path our it's running into some issue with reading the path, if on windows ensure the path is not too long for the project. We are working on better message handling in this section on a future SDK release.

Also if it still doesn't work could you try with the Unity version of this repository,  [Unity 2022.3.16f1](https://unity.com/download)? We weren't able to reproduce the issue with the released version 2022.3.42f1

**Comment by AlexThiv** [comment]
If you're still having this issue please reopen it with additional details.

**Comment by mludlum** [comment]
Apologies for the delay in responding. 
1.  I believe I installed lfs before cloning because when I tried installing again, it was already there.
2.  Cloned project again and used build 2022.3.16f1.  Project built/deployed/ran on my Quest 3.  
3.  Went back to my other clone with the 2022.3.42f1 build.  Deleted the library folder and then it built/deployed/ran.  

So perhaps I did something out of order originally?  

At first glance, the demo looks very cool.  Thank-you for your support on this.
